frontend/src/scenes/billing/billingLogic.ts

```diff
@@ -71,4 +111,9 @@ export const billingLogic = kea<billingLogicType<PlanInterface, BillingSubscript
             }
         },
     }),
+    urlToAction: ({ actions }) => ({
+        '*': ({ _: urlPath }: { _: string }) => {
+            actions.setUrlPath(urlPath)
+        },
+    }),
```

Comment:
Also as far as I understand this wouldn't be necessary if comparing against current scene.